src/octoprint/static/js/app/viewmodels/temperature.js

```diff
@@ -520,7 +497,7 @@ $(function () {
                 }
 
                 return {
-                    color: self.marking_props[mark.type].color,
+                    color: "#aeb6bf",
```

Comment:
I think this is where you need adjust the code to have the line the same color? You're forcing the gray.

Reply:
tl;dr this ended with me going "oh right, you can extract CSS properties via JS!" :)  Fixed to match the corresponding CSS.

![image](https://user-images.githubusercontent.com/704589/196295783-e63f1c51-1cb4-4bd4-8203-4e228bf82bbf.png)